code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -359,4 +386,19 @@
 	on = TRUE
 	icon_state = "thermo_base_1"
 
+/obj/machinery/atmospherics/components/unary/thermomachine/anomalous
+	allows_cooling = FALSE
+	has_pyro_anomaly_core = TRUE
+	name = "Anomalous temperature control unit"
+	desc = "Heats gas in connected pipes."
```

Comment:
These should be at the top.

Reply:
So I should move `/obj/machinery/atmospherics/components/unary/thermomachine/anomalous` to the top of the file?

Reply:
Fixed.